ClientTester/Program.cs

```diff
@@ -9,6 +13,8 @@ class Program
     {
         private static readonly string DEFAULT_IP_ADDRESS = "127.0.0.1";
 
+        private static Vector3 clientPos = new Vector3(-50f, -2f, -38f);
+
         static void Main(string[] args)
```

Comment:
I recommend rethinking the approach here. Instead of doing the command processing in this one function, look at enhancing reusability and maintainability. Consider building out a console "Command Processor" that can take and process arbitrary console "Commands". This way you can build out client test commands and avoid overloading a single class or method.  The side benefit is that the application would get a console command processor that can be used in scenarios outside of testing.

A pattern that you may find useful is https://en.wikipedia.org/wiki/Command_pattern (or just look at how packet processing is done).